### PR TITLE
Fix spelling in README.md

### PR DESCRIPTION
**Pull request by zurk**
Fix spelling in `README.md` and also brackets for NCCL installation guide.

Not a pro in spelling but fix some obvious moments to make it more pretty to read.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,31 +187,31 @@ script.
...
+- `log_dir` - this is the path to the directory where the logs shall be saved
 - `--char-map ../datasets/fsns/fsns_char_map.json` - path to the char map for mapping classes to labels.
 - `--blank-label 0` - indicates that class 0 is the blank label
 - `-b <batch-size>` - set the batch size used for training
 
 ## FSNS Demo
 
 In case you only want to see how the model behaves on a given image, you can use the `fsns_demo.py` script.
-This script expects a trained model, an image and a char map an prints you the predicted words in the
+This script expects a trained model, an image and a char map prints you the predicted words in the
```
Comment (Bartzi) [inline README.md]:
```suggestion
This script expects a trained model, an image and a char map and prints you the predicted words in the
```

File: README.md [hunk 2]
```diff
@@ -233,35 +233,35 @@ After you are done with preparing the dataset, you can start training.
...
+- `log_dir` - this is the path to the directory where the logs shall be saved
 - `--char-map ../datasets/textrec/ctc_char_map.json` - path to the char map for mapping classes to labels.
 - `--blank-label 0` - indicates that class 0 is the blank label
 - `-b <batch-size>` - set the batch size used for training
 
 ## Text Recognition Demo
 
 Analog to the `fsns_demo.py` script, we offer a demo script for text recognition named `text_recognition_demo.py`.
-This script expects a trained model, an image and a char map an prints you the predicted words in the
+This script expects a trained model, an image and a char map prints you the predicted words in the
```
Comment (Bartzi) [inline README.md]:
```suggestion
This script expects a trained model, an image and a char map and prints you the predicted words in the
```

**Comment by Bartzi** [comment]
Nice, thanks =)

**Comment by zurk** [comment]
You are welcome. Really nice model, by the way.

**Pull request merged**